lib/vm/src/context.rs

```diff
@@ -74,50 +74,50 @@ pub fn do_human_address<A: Api>(api: A, ctx: &mut Ctx, canonical_ptr: u32, human
 
 /** context data **/
 
-struct ContextData<T: Storage> {
-    data: Option<T>,
+struct ContextData<S: Storage> {
```

Comment:
Yeah, good rename. Generally in rust, all single generics are named `T` by convention. I found <T, U, V> a bit confusing, rather use `S`, `A`, etc in the code to make it clearer. Only did that when there were multi-args, but nice here for consistency.